Rakefile

```diff
@@ -9,12 +9,16 @@ Rake::TestTask.new(:test) do |t|
   end
 end
 
-task :default => [:test, :stdlib_test]
+task :default => [:test, :stdlib_test, :rubocop]
 
 task :stdlib_test do
   sh "ruby bin/test_runner.rb"
 end
 
+task :rubocop do
+  sh "rubocop -P"
```

Comment:
Personally I prefer the long option, `--parallel`,  to `-P` in a committed file.
Because it is explicit.

I also like the short option on a console, because it needs fewer types.
But typing count is not important in a committed file. Readable code is more important.

Reply:
Totally! I updated the option.